translations/ui-plugin-find-contact/en.json

```diff
@@ -2,7 +2,9 @@
   "button.addContact": "Add contact",
   "meta.title": "Add (assign) 'contact person' to organization (plugin)",
   "meta.pluginTitle": "Contacts",
-  "button.close": "Close",
+  "button.cancel": "Cancel",
+  "button.save": "Save",
+  "totalSelected": "Total selected: ",
```

Comment:
`: ` shouldn't be in the translation, I suppose

Reply:
Yes, good point. Fixed